compiler/rustc_next_trait_solver/src/solve/eval_ctxt/mod.rs

```diff
@@ -930,6 +930,10 @@ where
             && goal.param_env.visit_with(&mut visitor).is_continue()
     }
 
+    pub(super) fn sub_ty_vids_raw(&self, a: ty::TyVid, b: ty::TyVid) {
```

Comment:
renaming to something that doesnt sound "directional". we only care about whether two vars are related to eachother *in some way*, but not whether its `a <: b` or `b <: a`. maybe "sub_unify_ty_vids_raw` or smth idk :3

Reply:
`sub_relate_X`